ui/src/hooks/beta/useComputedPlaybackFields.tsx

```
@@ -0,0 +1,98 @@
+    playlistTimestamp: firstPlaylistTimestamp ?? "",
+    // enabled: needsOffsets,
+  });
+
+  const stream = feedStreams?.feedStreams.results[0];
+
+  const playlistTimestamp = firstPlaylistTimestamp
+    ? parseInt(firstPlaylistTimestamp)
+    : 0;
+
+  const startOffset =
+    stream && stream.startTime && startTimestamp && startTimestamp !== ""
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)